packages/earthdata-login-client/README.md

```diff
@@ -0,0 +1,27 @@
+# @cumulus/launchpad-auth
+
+Utilities for OAuth authentication using [NASA Earthdata Login](https://urs.earthdata.nasa.gov/).
+
+## Versioning
+
+Cumulus uses a modified semantic versioning scheme and minor releases likely include breaking changes.
+
+Before upgrade, please read the Cumulus [release notes](https://github.com/nasa/cumulus/releases) before upgraded.
+
+It is strongly recommended you do not use `^` in your `package.json` to automatically update to new minor versions. Instead, pin the version or use `~` to automatically update to new patch versions.
```

Comment:
Didn't we recently agree to wrap at 80 columns for Markdown? Perhaps you weren't on the call or had dropped off? @yjpa7145 had brought this up.